site/en/blog/chrome-108-beta/index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Chrome 108 beta
+description: >
+  New CSS viewport units, Federated Credential Management API, Variable COLRv1 fonts, and more.
+subhead: >
+  New CSS viewport units, Federated Credential Management API, Variable COLRv1 fonts, and more.
+layout: 'layouts/blog-post.njk'
+date: 2022-10-27
+hero: 'image/kheDArv5csY6rvQUJDbWRscckLr1/FbdknipjOqHfVYEbvcLz.png'
+alt: >
+  Chrome 108 beta hero logo
+tags:
+  - beta
+  - chrome-108
+---
+
+Unless otherwise noted, changes described below apply to the newest Chrome beta channel release for Android, ChromeOS, Linux, macOS, and Windows. Learn more about the features listed here through the provided links or from the list on ChromeStatus.com. Chrome 108 is beta as of October 27, 2022. You can download the latest on [Google.com](https://www.google.com/chrome/beta/) for desktop or on Google Play Store on Android.
+
+## CSS
+
+Chrome 108 includes a number of new CSS features.
+
+### CSS Overflow for replaced elements
+
+Chrome will start to roll out a change that allows developers to use the existing `overflow` property with replaced elements that paint outside the content-box. Paired with `object-view-box` this can be used to create an image with a custom glow or shadow applied, with proper ink-overflow behavior like a CSS shadow would have. 
+
+This is a potential breaking change, find out more in [A change to overflow on replaced elements](/blog/overflow-replaced-elements/).
+
+### Small, Large, Dynamic, and Logical viewport units
+
+This adds support for small (`svw`, `svh`, `svi`, `svb`, `svmin`, `svmax`), large (`lvw`, `lvh`, `lvi`, `lvb`, `lvmin`, `lvmax`), dynamic (`dvw`, `dvh`, `dvi`, `dvb`, `dvmin`, `dvmax`), and logical (`vi`, `vb`) units.
+
+### CSS `break-after`, `break-before` and `break-inside` support
+
+Support for the avoid value of the CSS fragmentation properties `break-before`, `break-after`, and `break-inside` when printing. This value tells the browser to avoid breaking before, after or inside the element it is applied to. For example, the following CSS avoids a figure being broken at a page break.
+
+```css
+figure {
+    break-inside: avoid;
+}
+```
+
+This feature has been added due to Chrome 108 adding support for LayoutNG printing.
+
+### Last baseline item alignment
+
+This feature allows developers to align items within either flex or grid layout by their last baseline, instead of their first. This is done via the following properties: 
+
+- `align-items: last baseline;`
+- `justify-items: last baseline;` 
+- `align-self: last baseline;`
+- `justify-self: last baseline;`
+
+## `ContentVisibilityAutoStateChanged` event
```

Comment:
Isn't the event name all-lowercase?

Reply:
I took the data from Chrome Status where it is uppercase, so if that's the case then please fix it there: https://chromestatus.com/feature/5179553291436032